public/index.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#21262F">
+	
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.png">
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-icon.png">
+
+    <meta property="og:image" content="%PUBLIC_URL%/social-image.png">
+    <meta property="og:image:type" content="image/png">
+    <meta property="og:image:width" content="1275">
+    <meta property="og:image:height" content="697">
+
+    <meta property="og:title" content="Impact Development – Customize your Game">
+    <meta property="og:site_name" content="Impact Development">
+    <meta property="og:url" content="https://impactdevelopment.io">
+    <meta property="og:description" content="Customize your game with the Impact Client for Minecraft">
+
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:description" content="Customize your game with the Impact Client for Minecraft">
+    <meta name="twitter:title" content="Impact Development">
+    <meta name="twitter:site" content="@ImpactDevelopment">
+    <meta name="twitter:image" content="%PUBLIC_URL%/social-image.png"}>
+	<meta name="author" content="Cody Bennett">
+
+    <title>Impact Development</title>
+  </head>
+  <body>
+    <noscript>
+      You need to enable JavaScript to run this app.
+    </noscript>
```

Comment:
This is unacceptable. JS should improve the experience not be a minimum requirement.

Reply:
The ad service requires JS, and so does the handling of operating system detection in the website currently.